### ISSUE TITLE
After the migration the dependencies appear have a yellow warning

### ISSUE THREAD

**Issue by csunkara-ipipeline** [CodeFactoryLLC/WebForms2BlazorServer#7]
After the migration the dependencies appear to have disappeared. I am trying to see why has that happened now and how to reverse it. I originally thought the issue was gone after I tried it on a new project but it is coming back once I closed the solution and opened it back.

Just a note that I added the Blazor project for migration into an existing solution that hosts the WebForms website project.

![image](https://user-images.githubusercontent.com/63052952/107743021-0bcbbe80-6d08-11eb-83e5-254bff4a5629.png)

**Comment by jhannahLCD** [comment]
@csunkara-ipipeline This is on a fresh project, right?  Do they appear after a build (some of those are nuget-backed and won't actually bring down their payloads until a Build event on the project happens).

**Comment by csunkara-ipipeline** [comment]
@jhannahLCD Yes, this was a fresh project. I tried building, cleaning and rebuilding etc. and none clear the issue.

**Comment by jhannahLCD** [comment]
@csunkara-ipipeline reaching out via email to get a few more details about your project.
